packages/sanity/package.json

```diff
@@ -127,6 +127,7 @@
   "scripts": {
     "build": "pkg-utils build --strict --check --clean",
     "build:bundle": "vite build --config package.bundle.ts",
+    "build:web-workers": "vite build --config package.worker.ts",
```

Comment:
This is very dirty, and will be replaced by first-class support for Web Worker bundling in `@sanity/pkg-utils`: https://github.com/sanity-io/pkg-utils/issues/987